lib/logstorage: fix field_max() returning the value at min source field instead of max (#1295)
VictoriaMetrics/VictoriaLogs: docs/victorialogs/CHANGELOG.md

* FEATURE: [LogsLQ](https://docs.victoriametrics.com/victorialogs/logsql/): add an ability to search across multiple log fields with the `field_name_prefix*:filter` syntax. See [these docs](https://docs.victoriametrics.com/victorialogs/logsql/#searching-over-multiple-fields) for details. See [#82](https://github.com/VictoriaMetrics/VictoriaLogs/issues/82).
* FEATURE: [Loki data ingestion](https://docs.victoriametrics.com/victorialogs/data-ingestion/promtail/): Add an ability to add the given prefix to log field names parsed from JSON message. This may be used for avoiding clash between log stream labels and the automatically parsed log field names. See [these docs](https://docs.victoriametrics.com/victorialogs/data-ingestion/promtail/#parsing-log-message) for details. See [#1127](https://github.com/VictoriaMetrics/VictoriaLogs/issues/1127).

> * BUGFIX: [LogsQL](https://docs.victoriametrics.com/victorialogs/logsql/): fix [`field_max` stats function](https://docs.victoriametrics.com/victorialogs/logsql/#field_max-stats) returning the field value at the row with the minimum source field value instead of the maximum. See [#1294](https://github.com/VictoriaMetrics/VictoriaLogs/issues/1294).

Reviewer: ```suggestion
* BUGFIX: [LogsQL](https://docs.victoriametrics.com/victorialogs/logsql/): properly return the field value for the row with the maximum `_time` field from [`field_max(_time, field)`](https://docs.victoriametrics.com/victorialogs/logsql/#field_max-stats). See [#1294](https://github.com/VictoriaMetrics/VictoriaLogs/issues/1294).
```
